### PR TITLE
Add dict_replace_if_equal command to buildozer.

### PR DESCRIPTION
**Pull request by tsell**
This is useful for making modifications at scale, where you only want to replace certain dictionary values. To illustrate, suppose you have a BUILD file containing many targets, some of which use some "old_value" which you wish to replace with "new_value" in a dictionary, and some of which use other values that you don't want to modify:

```
macro(
  name = "foo",
  macro_values = {
    "key": "old_value",
  },
)

macro(
  name = "bar",
  macro_values = {
    "key": "neutral_value",
  },
)
```

With this new command, these can all be modified like so:
```
buildozer "dict_replace_if_equal macro_values key old_value new_value" //path/to/these/targets:*
```

Resulting in the desired change:
```
macro(
  name = "foo",
  macro_values = {
    "key": "new_value",
  },
)

macro(
  name = "bar",
  macro_values = {
    "key": "neutral_value",
  },
)
```

### CODE REVIEW

**Review by vladmos — CHANGES_REQUESTED** [review]
Thanks!

File: edit/buildozer.go [hunk 1]
```diff
@@ -745,6 +745,35 @@ func cmdDictRemove(opts *Options, env CmdEnvironment) (*build.File, error) {
...
+	if !ok {
+		return env.File, nil
+	}
+
+	prev := DictionaryGet(dictAttr, key)
+	if prev == nil {
+		return nil, fmt.Errorf("key '%s' not found in dict", key)
+	}
+	if e, ok := prev.(*build.StringExpr); ok {
+		if labels.Equal(e.Value, oldV, env.Pkg) || e.Value == oldV {
```
[the remark this answers is not in the document]
Comment (tsell) [reply edit/buildozer.go]: Done.

File: edit/buildozer_test.go [hunk 2]
```diff
@@ -351,6 +351,122 @@ func TestCmdDictListAdd(t *testing.T) {
...
+		}
+		got := strings.TrimSpace(string(build.Format(bld)))
+		expected := strings.TrimSpace(string(build.Format(expectedBld)))
+		if got != expected {
+			t.Errorf("cmdDictReplaceIfEqual(%d):\ngot:\n%s\nexpected:\n%s", i, got, tt.expected)
+		}
+	}
+}
+
+
```
[the remark this answers is not in the document]
Comment (tsell) [reply edit/buildozer_test.go]: Done.

**Review by tsell — COMMENTED** [review]
Done

**Review by vladmos — APPROVED** [review]

**Pull request merged**